source/devices/AM62X/linux/Release_Specific_Supported_Platforms_and_Versions.rst

```diff
@@ -19,10 +19,18 @@ Linux SDK.
 
 The following platforms and EVMs are supported with Processor SDK.
 
-+--------------+-----------+-----------+-----------------------+-------------------+------------------+------------------+
-| **Platform** | **EVM**   | **Tested  | **Document**          | **Processor SDK   | **Processor SDK  | **Processor SDK  |
-|              |           | Version** |                       | Linux**           | RT Linux**       | RTOS**           |
-+--------------+-----------+-----------+-----------------------+-------------------+------------------+------------------+
-| AM62x-SK     | AM62x-SK  | E2        | Hardware User's Guide | Y                 | Y                | Y                |
-+--------------+-----------+-----------+-----------------------+-------------------+------------------+------------------+
++--------------+-------------+--------------------------------------------------------------------+-------------------+------------------+
+| **Platform** | **EVM**     | **Document**                                                       | **Processor SDK   | **Processor SDK  |
+|              |             |                                                                    | Linux**           | RT Linux**       |
++--------------+-------------+--------------------------------------------------------------------+-------------------+------------------+
+| AM62x        | SK-AM62     | `User's Guide <https://www.ti.com/tool/SK-AM62#tech-docs>`__       | Y                 | Y                |
++--------------+-------------+--------------------------------------------------------------------+-------------------+------------------+
+| AM62x        | SK-AM62B    | `User's Guide <https://www.ti.com/tool/SK-AM62B#tech-docs>`__      | Y                 | Y                |
++--------------+-------------+--------------------------------------------------------------------+-------------------+------------------+
+| AM62x        | SK-AM62B-P1 | `User's Guide <https://www.ti.com/tool/SK-AM62B-P1#tech-docs>`__   | Y                 | Y                |
++--------------+-------------+--------------------------------------------------------------------+-------------------+------------------+
+| AM62x        | SK-AM62-LP  | `User's Guide <https://www.ti.com/tool/SK-AM62-LP#tech-docs>`__    | Y                 | Y                |
++--------------+-------------+--------------------------------------------------------------------+-------------------+------------------+
+| AM62x        | BEAGLE-PLAY | `User's Guide <https://www.ti.com/tool/BEAGL-PLAY-SBC#tech-docs>`__| Y                 | Y                |
```

Comment:
I don't like the text running into the table bounds here, but the linter didn't complain about a malformed table, so I guess this is allowed.